.github/dependabot.yaml

```diff
@@ -33,11 +33,7 @@ updates:
         update-types:
           - minor
           - patch
-    # only update internal github actions, external github actions are handled
-    # by https://github.com/hashicorp/security-tsccr/tree/main/automation
-    allow:
-      - dependency-name: "hashicorp/*"
   - package-ecosystem: "docker"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "daily"
```

Comment:
The change from `weekly` -> `daily` is just to give us the best chance to pass Red Hat's security scan on the UBI images, since we don't have a way to do that scan until we submit them at the end of a release.